libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```
@@ -80,17 +87,17 @@ $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
     .wrapper {
       position: relative;
       display: inline-flex;
-      align-items: center;
+      align-items: flex-start;
       vertical-align: top;
       padding-right: $spacing-to-label; // Ensure same padding on right side of label
-      min-height: map-get($checkbox-radio-sizes, 'md');
+      @include wrapper-top-bottom-padding(map-get($checkbox-radio-sizes, 'md'));
```

Consider inlining mixin for readability (and move above `padding-right`):
```suggestion
      padding: (map-get($checkbox-radio-sizes, 'md') - $checkbox-icon-size) / 2;
```

---

Resolved by ee8237b